tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -73,3 +73,7 @@ spec:
     env:
     - name: ENABLE_EXEC_SERVICE
       value: "false"
+  - name: upload-results
+    image: amazon/aws-cli
+    workingDir: $(workspaces.results.path)
+    command: ["aws s3 cp . s3://$(params.results-bucket)/$(date +'%Y/%m/%d/%HH/%mm')  --recursive "]
```

Comment:
Can we include seconds in this timestamp? Or perhaps a uuid? I can see this colliding otherwise.